{
    try {
        console.log(x);
    } catch (e) {
        console.log(e.message);
    }

    
    let x = 5;
    console.log(x);
}
